Client/WarFare/GameProcMain.cpp

```diff
@@ -2269,6 +2288,13 @@ bool CGameProcMain::MsgRecv_Regen(Packet& pkt)
 	return true;
 }
 
+void CGameProcMain::MsgRecv_FogDensity(Packet& pkt)
+{
+	int16_t iValue = pkt.read<int16_t>();
+
+	s_pEng->CameraGetActive()->m_fFogDensity = iValue * 0.01f;
```

Comment:
This really should be using a setter. :)

Reply:
Many things here needs to get refactored with getters and setter. But as I'm not very familiar with the source code and doesn't have a big view of the whole project, I'm just fitting my self to the coding convention/style they used there, even though we know there is a lot to change and improve.
But anyways, added a setter :)

Reply:
That is true, but we don't want to be adding to these (many) cases. Just removing them slowly :)